detox/e2e/test/autocomplete/edit_post.e2e.js

```diff
@@ -22,15 +27,27 @@ describe('Autocomplete', () => {
         // # Tap the send button
         await element(by.id('send_button')).tap();
 
+        // Explicitly wait on Android before verifying error message
+        if (isAndroid()) {
+            await wait(timeouts.ONE_SEC);
+        }
+
         // # Open edit screen
         await element(by.text(message)).longPress();
-        await element(by.text('Edit')).tap();
+
+        const slide = element(by.id('slide_up_panel'));
+        await slide.swipe('up');
+
+        const edit = element(by.text('Edit'));
+        await edit.tap();
 
         // # Open autocomplete
         await expect(element(by.id('autocomplete.at_mention.list'))).not.toExist();
         await element(by.id('edit_post.input')).typeText(' @');
 
         // * Expect at_mention autocomplete to render
         await expect(element(by.id('autocomplete.at_mention.list'))).toExist();
+
+        await element(by.id('edit-post.close')).tap();
```

Comment:
This is failing for me. Please fix.

```suggestion
        await element(by.id('edit_post.close')).tap();
```